### ISSUE TITLE
Discussion: Repo management ideas

### ISSUE THREAD

**Issue by aguynamedben** [rt2zz/redux-persist#943]
We're trying to manage this project better to develop a strong community around this repo. There are 308 open issues and 30 open pull requests right now, and @rt2zz and me are gradually working through the backlog as we can.

I wanted to gather ideas on how we can better manage this important project.

Ideas:
* Use bots better
  * stale bot to auto-expire issues if they haven't been active for a certain amount of time ([example](https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues/1823#issuecomment-445910474))
  * rennovate bot to keep dependencies updates ([example](https://github.com/electron-react-boilerplate/electron-react-boilerplate/commit/e1b41f60dfa9752dfa7e8f017a8d21a5c4655f86))
  * Review how CI feedback on GitHub issues (if any) works or can be improved
* Manage the issue backlog better
  * Better use of GitHub labels ([one example](https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues))
    * "Good first issue" label for new contributors
  * Review and improve the project's GitHub issue templates
  * Auto-ask for an example repo if someone is experiencing a bug
* Encourage new and repeat contributions
  * Double check the "Contributor" badge to make sure people with PRs are rewarded in a simple way
  * Maybe send a simple token of appreciation to people who contribute - i.e. mail a postcard, sticker, etc.

This is not an exhaustive or definite list, but a start to a discussion. Please respond with more ideas if you have any!

**Comment by dereknelson** [comment]
I would love a #redux-persist reactiflux channel!

Maybe we we could highlight known issues? I spent a while combing through documentation trying to figure out why my state refused to persist on android - it was only after reading through a number of different open issues was I made aware that persisting stores larger than a certain size just won't cooperate with AsyncStorage on android. If I'd read that somewhere I would've either
A. been way more careful with what I persist
B. fixed it by using one of the workarounds out there way sooner 
but it was the kind of error that fails silently in a way

**Comment by aguynamedben** [comment]
@dereknelson I created a Discord channel for #redux-persist: https://github.com/rt2zz/redux-persist#chat-room I'd appreciate if you'd join and help me get the ball rolling!

**Comment by aguynamedben** [comment]
@rt2zz Can you install this stale bot for redux-persist? https://github.com/apps/stale

Nothing will happen until the file `.github/stale.yml` is created. I'll dig into the settings and configure appropriately. Probably 30 or 60 days after an issue has gone dormant, the bot will comment, then 7 days after that if nobody has replied, the issue will be closed with `wontfix`. Issues are auto-reopened if somebody replies.

They currently use this at [electron-react-boilerplate](https://github.com/electron-react-boilerplate/electron-react-boilerplate) and it seems efficient. I think it will be a good way for us to whittle the 304 issues down to the "real" remaining issues.

**Comment by nmay231** [comment]
@rt2zz  @aguynamedben If you are looking for help, I could possibly work on labeling issues/reviewing PRs.

I'm currently helping with redux-undo, but I could push some time towards this.
